ichnaea/service/search/schema.py

```diff
@@ -38,9 +38,16 @@ class WifisSchema(SequenceSchema):
     wifi = WifiSchema()
 
 
+class FallbackSchema(MappingSchema):
+
+    lacf = SchemaNode(Integer(), missing=None)
+    ipf = SchemaNode(Integer(), missing=None)
+
+
 class SearchSchema(MappingSchema):
 
     radio = SchemaNode(String(),
                        validator=OneOf(RADIO_STRINGS), missing=None)
     cell = CellsSchema(missing=())
     wifi = WifisSchema(missing=())
+    fallbacks = FallbackSchema(missing={})
```

Comment:
And sometimes I wish Python had an immutable dict type, so you could actually specify a method level or class level dict-like default argument. Using a tuple as a placeholder for list works, but isn't ideal either.